### DISCUSSION TITLE
Open URLs with external browser

### DISCUSSION THREAD

**Discussion by frapox** [martinrotter/rssguard#1502] [Q&A] [answered]
Maybe I'm blind but I cannot find an option or setting to open URLs of posts with external browser by default (with one click, not clicking right and then choosing the "open with..."). Any suggestion?

**Comment by Ac314** [comment, marked as answer]
Network & web & tools -> External web browser -> Always open hyperlinks in external web browser.

The simplest method - just double click on the article in the article list. 

Or in the article itself - look for "URL" in the title:
![image](https://github.com/user-attachments/assets/5f7449e7-f3b5-4c02-a841-b62ca83247d8)

**Reply by frapox** [reply]
@Ac314 thank you very much!

**Reply by larsen0815** [reply]
You can also define a shortcut to use the Enter key:
![Clipboard-1](https://github.com/user-attachments/assets/2db21b3d-68be-404b-b869-9545d4769642)
